Initial commit for the tdk frontend application

ajayan508 opened the pull request:
First checkin for the TDK Front end application

ajayan508 commented:
@mhughesacn We will check and  rectify all these issues

mhughesacn commented:
Hi @ajayan508 : Thank you for the comprehensive answer.  The Pixabay image needs to be handled differently.  Look at how gerrit tdk does it.  It needs to be in its own directory with a different LICENSE file.  NOTICE refers to the directory (3rd para in tdk land NOTICE).  This approach was agreed with Comcast Legal.  As the excel image was loaded in 2021, the pixabay license in TDK is OK to use.

ajayan508 commented:
> Hi @ajayan508 : Thank you for the comprehensive answer. The Pixabay image needs to be handled differently. Look at how gerrit tdk does it. It needs to be in its own directory with a different LICENSE file. NOTICE refers to the directory (3rd para in tdk land NOTICE). This approach was agreed with Comcast Legal. As the excel image was loaded in 2021, the pixabay license in TDK is OK to use.

Hi @mhughesacn , I have added the LICENSE file and modified NOTICE as per your suggestions, Could you please check if this is OK
